packages/shared/src/components/tooltips/BaseTooltip.tsx

```diff
@@ -51,14 +58,20 @@ export function BaseTooltip(
       onHide={onHide}
       delay={delay}
       allowHTML
+      animation={!(disableInAnimation && disableOutAnimation)}
```

Comment:
It can't ever hit this outAnimation because you return if it exists.

Reply:
Yes, it won't but this is for tippy's internal delay for animation which has to be applied here as well.

Reply:
That return was to avoid our custom animation.